contracts/Vault.vy

```diff
@@ -1401,14 +1401,14 @@ def expectedReturn(strategy: address = msg.sender) -> uint256:
 def _reportLoss(strategy: address, _loss: uint256):
```

Comment:
Can you instead make the `_loss` parameter to `loss` instead?

Reply:
fixed in latest